engine/installation/linux/docker-ee/centos.md

```diff
@@ -14,20 +14,22 @@ title: Get Docker EE for CentOS
 
 {% include ee-linux-install-reuse.md section="ee-install-intro" %}
 
-## Prerequisites
+## Install Docker EE on CentOS
```

Comment:
Why? It gets added automatically from the `page.title` variable in the frontmatter.